server/modules/turf/turf.controller.ts

```
@@ -146,99 +176,133 @@
       } catch (error) {
         next(error);
       }
-    }
+    },
   );
 
   private async validateAndParseUpdateData(
     data: any,
-    next: NextFunction
+    next: NextFunction,
   ): Promise<Partial<ITurf> | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)